.circleci/config.yml

```diff
@@ -112,14 +112,47 @@ jobs:
       - store_artifacts:
           path: /root/sd/junit
 
+  tests:
+    machine: true
+    environment:
+      DOCKER_API_VERSION=1.23
+    parallelism: 3
+    steps:
+      - run: sudo apt-get install -y make
+      - checkout
+
+      - run:
+          name: Build Docker images
+          command: |
+            set +o pipefail
+            docker images
+            fromtag=$(docker images |grep securedrop-test |head -n1 |awk '{print $2}')
+            cd securedrop && EXTRA_BUILD_ARGS="--cache-from securedrop-test:${fromtag}" make images TAG=${CIRCLE_SHA1}
```

Comment:
We might want  to do `${fromtag:?}` to have to puke if the above `grep|head|awk` returns nothing.

Reply:
I think it's better to build without cache than to get into a stuck state if circle clears caches or cache population/loading fails in some manner.